### PR TITLE
Add Menu API and MenuItem API cases in nw.js project.

### PR DESCRIPTION
**Pull request by xiaoyu3x**
relative Menu API:
  menu.items
  menu.popup
  menu.removeAt
  menu.removeItem
relative MenuItem API:
  item.checked
  item.enabled
  item.icon
  item.label
  item.type

| | Pass | Fail | Block|
---|---|---|---
Total |  14 | 0 | 0
Auto|  0 | 0 | 0
Manual | 14 | 0| 0

### CODE REVIEW

File: nwjs-iot/nwjsapi/menu/menu-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+<meta charset="utf-8">
+<title>Menu Popup</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="author" title="zhang xiaoyux" href="mailto:xiaoyux.zhang@intel.com">
+<link rel="help" href="http://docs.nwjs.io/en/latest/References/Menu/#menupopupx-y">
+<p>
+  <strong>Test Steps:</strong>
+</p>
+<ol>
+  <li>Click the right mouse button on the "CONTEXTMENU" text.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/menu/menu-popup/index.html]: Right click on the "CONTEXTMENU" text.

Comment (Honry) [inline nwjs-iot/nwjsapi/menu/menu-popup/index.html]: Is there really a "CONTEXTMENU" text?

Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/menu/menu-popup/index.html]: To "Is there really a "CONTEXTMENU" text?"

Sorry,
  I forgot  to add "CONTEXTMENU" text as <p> value.
Has added code:
`<p>CONTEXTMENU</p>`

To "Right click on the "CONTEXTMENU" text."

Furthermore test steps has been changed:
`<li>Right click on the "CONTEXTMENU" text, then click any item in drop down list.</li>`

**Comment by Honry** [comment]
LGTM

**Pull request merged**